src/misc/setup.cpp

```diff
@@ -1205,6 +1205,8 @@ bool Config::WriteConfig(const std_fs::path& path) const
 				// before writing them into the config.
 				auto s = format_str(help);
 
+				s = strip_ansi_markup(s);
```

Comment:
Why is this needed? It's a functional change stuck in a refactor commit.

Reply:
Removed.